account_banking_mandate/models/account_banking_mandate.py

```diff
@@ -183,7 +183,7 @@ def _check_valid_state(self):
 
     @api.model
     def create(self, vals=None):
-        if vals.get('unique_mandate_reference', 'New') == 'New':
+        if not vals.get('unique_mandate_reference'):
```

Comment:
@ThomasBinsfeld if the user provides `New` I think you must preserve the current behaviour.

Reply:
Done